frontend/src/queries/schema/schema-surveys.ts

```diff
@@ -37,6 +37,7 @@ export type SurveyQuestionBranchingType = 'next_question' | 'end' | 'response_ba
 export interface SurveyQuestionSchema {
     type: SurveyQuestionType
     question: string
+    id?: string
```

Comment:
@lucasheriques any reason you did not the id to the schemas?

Reply:
no reason in particular. they are just not needed, as we only use them for ordering in the events

Reply:
we do `question.get("id") at least in 2 ocasions so I'd say this should be part of the schema